src/views/Ifos/components/IfoCardV1Data.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import useGetPublicIfoV1Data from 'hooks/ifo/v1/useGetPublicIfoData'
+import useGetWalletIfoV1Data from 'hooks/ifo/v1/useGetWalletIfoData'
+import { Ifo } from 'config/constants/types'
+import IfoFoldableCard from './IfoFoldableCard'
+
+interface Props {
+  ifo: Ifo
+}
+
+const IfoCardV1Data: React.FC<Props> = ({ ifo }) => {
+  const publicIfoData = useGetPublicIfoV1Data(ifo)
+  const walletIfoData = useGetWalletIfoV1Data(ifo)
+
+  return (
+    <IfoFoldableCard ifo={ifo} publicIfoData={publicIfoData} walletIfoData={walletIfoData} isInitiallyVisible={false} />
+  )
+}
+
+export default IfoCardV1Data
```

Comment:
Should this file be removed as it is V1?

Reply:
No, this is a new component. The card is generic but we need a wrapper around the card to fetch the right data (V1 or V2)